### DISCUSSION TITLE
[Feature Request] Element Zapper to context menu option

### DISCUSSION THREAD

**Discussion by qedda** [uBlockOrigin/uBlock-issues#2821] [Ideas] [answered]
As the title says, an option to add Element Zapper to the context menu.
Currently you can only enable/disable Element Picker on the context menu, however I would appreciate it if you could add an option to add the Zapper also. 

I use the Zapper a bit more than the picker since I tend to temporarily disable stuff on certain sites, this is mostly because some sites tend to break with permanent options enabled. 
Also I love the Zapper it's too handy not to have as an option. 

PS. I know there's keyboard shortcuts but I'd rather have a mouse only option (aka context menu), if possible. 

Kind regards,
Qedda

**Comment by krystian3w** [comment, marked as answer]
https://github.com/uBlockOrigin/uBlock-issues/issues/3882
